src/components/Form.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { IHandleTodoInsert } from './Interface';
+
+interface ITodoInsertProps {
+  handleTodoInsert: IHandleTodoInsert;
+}
+
+const Form = ({ handleTodoInsert }: ITodoInsertProps) => {
+  const [input, setInput] = useState<string>('');
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInput(e.target.value);
+  };
+
+  const handleFormSubmit = (e: React.FormEvent) => {
```

Comment:
저도 멘토님께서 짚어주신 부분인데 해당 부분은 이벤트 대상을 명시해주는게 좋은것 같습니다.

```suggestion
  const handleFormSubmit = (e: React.FormEvent<HTMLFormElement>) => {
```

Reply:
생각하지 못한 부분인데 알려주셔서 감사합니다!! :D